cmd/otel-allocator/watcher/promOperator_test.go

```diff
@@ -946,7 +1080,7 @@ func TestNamespaceLabelUpdate(t *testing.T) {
 
 	select {
 	case <-events:
-	case <-time.After(time.Second):
+	case <-time.After(3 * time.Second):
```

Comment:
Can we standardize on 5 seconds for these kinds of tests? They usually complete much faster, and having to wait 5 seconds for a fail is a fair price to pay for lack of random flaky failures on GHA runners.